content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -31,11 +31,11 @@ The Istio data plane components, the Envoy proxies, handle data flowing through
 the system. The Istio control plane component, Istiod, configures
 the data plane. The data plane and control plane have distinct performance concerns.
 
-## Performance summary for Istio {{< istio_release_name >}}
```

Comment:
This need not change, right?

Reply:
And I see looking further we have done the others.